api/v1alpha1/oidc_types.go

```diff
@@ -48,8 +49,49 @@ type OIDC struct {
 	RedirectURL *string `json:"redirectURL,omitempty"`
 
 	// The path to log a user out, clearing their credential cookies.
+	//
 	// If not specified, uses a default logout path "/logout"
 	LogoutPath *string `json:"logoutPath,omitempty"`
+
+	// ForwardAccessToken indicates whether the Envoy should forward the access token
+	// via the Authorization header Bearer scheme to the upstream.
+	// If not specified, defaults to false.
+	// +optional
+	// +notImplementedHide
+	ForwardAccessToken *bool `json:"ForwardAccessToken,omitempty"`
+
+	// DefaultTokenExpireIn is the default lifetime of the id token and access token.
+	// Please note that Envoy will always use the expiry time from the response
+	// of the authorization server if it is provided. This field is only used when
+	// the expiry time is not provided by the authorization.
+	//
+	// If not specified, defaults to 0. In this case, the "expires_in" field in
+	// the authorization response must be set by the authorization server, or the
+	// OAuth flow will fail.
+	//
+	// +optional
+	// +notImplementedHide
+	DefaultTokenExpireIn *metav1.Duration `json:"defaultTokenExpireIn,omitempty"`
```

Comment:
```suggestion
	DefaultTokenExpiresIn *metav1.Duration `json:"defaultTokenExpireIn,omitempty"`
```

Reply:
we need a better name here `DefaultTokenExpiresIn` that encapsulates default expiry duration for the token

Reply:
`DefaultTokenExpiryDuration`
`DefaultTokenTTL`

Reply:
+1 on `DefaultTokenTTL`